lib/src/main/kotlin/app/cash/nostrino/model/Kind.kt

```diff
@@ -0,0 +1,18 @@
+package app.cash.nostrino.model
+
+/**
+ * Represents the kind of an event as defined
+ * by the protocol: https://github.com/nostr-protocol/nips#event-kinds
+ */
+enum class Kind(val value: Int) {
```

Comment:
@jmateoac I made it mostly to clean up the content deserialiser, to avoid reaching into various types.

Happy with either a sealed interface, or maybe just a value type. What do you think?

Reply:
Looks like it's a little painful to serialise/deserialise sealed classes with Moshi, so I'll just use a value type instead.